spec/integration/foods_index_spec.rb

```diff
@@ -0,0 +1,53 @@
+# spec/features/food_list_spec.rb
```

Comment:
-  You have done great so far :+1: Your application looks great and works as expected but there are a few requirements you need to meet before moving to the next milestone :+1: 
- [ ] Looking at the screenshot below, you can see that about 3 tests are falling, These might be because of the changes you made from the last review, 
- These tests are there to make sure our code meets the requirements and functions as we expect, so while working on projects, I'll greatly advise you to always re-run your test suite anytime you make a major change to your codebase. On that note, I'll suggest you take a look at these tests and try to fix them. :+1: :smile: :small_airplane: 

- - Video reference:
![Screenshot from 2023-10-12 10-31-14](https://github.com/zuhebahmed88091/Recipe_app/assets/107632780/528a5481-1921-4a97-ac46-5e71583d28cf)

Reply:
Resolved